index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=poppins">
+    <link rel="stylesheet" href="./css/style.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <a class="logo" href="https://www.linkedin.com/in/jalal-ahmadi-58312b270/"><b>My Lgo</b></a>
+        <a href="#"><i class="material-icons">	list</i></a>
```

Comment:
- [ ] As requested by the previous reviewer in this [link](https://github.com/jalal1379/Microverse_portflaio/pull/1#discussion_r1214680793) you are to wrap the hamburger menu icon in a semantic `nav` tag becasue this is good html standard way of building your project. below is an example of how it should be implemented. :+1: 
```html
        <nav>
            <a href="#"><i class="material-icons">list</i></a>
        </nav>
```